src/app/screens/Unlock/index.jsx

```diff
@@ -61,9 +65,18 @@ function Unlock() {
           primary
           disabled={password === ""}
         />
+
+        <div className="flex justify-center space-x-1 mt-5">
+          <span className="text-text-gray-bitcoin ">Need help? Contact </span>
+          <TextButton
+            label="Alby Support"
+            primary
+            className="font-serif text-base"
+          />
+        </div>
       </form>
       {error && (
```

Comment:
In the new design I would place the error directly below the input field and give it a red color. (Feels out of place now below support.)

See for e.g. t
<img width="478" alt="Schermafbeelding 2021-12-16 om 11 14 08" src="https://user-images.githubusercontent.com/12894112/146352526-26bed2cf-8d19-4de9-885c-f23b2aa31e39.png">
he setting a password screen: